code/game/machinery/doors/airlock.dm

```diff
@@ -1123,7 +1124,10 @@
 	if( operating || welded || locked || seal )
 		return FALSE
 	if(!forced)
-		if(!hasPower() || wires.is_cut(WIRE_OPEN))
+		if(!hasPower())
+			return FALSE
+		wires.activate_wire(WIRE_OPEN)
+		if (wires.is_cut(WIRE_OPEN))
```

Comment:
uh what?

Reply:
This is correct, you want devices on the wire to get pulsed even when it is cut.